vignettes/customizing-dccvalidator.Rmd

```diff
@@ -40,6 +40,17 @@ To create a custom version of the app, you'll need to follow these steps:
    [Deploying dccvalidator](https://sage-bionetworks.github.io/dccvalidator/articles/deploying-dccvalidator.html) 
    vignette.
 
+Alternatively, instead of forking the repository you can install dccvalidator
+and create an `app.R` file containing the following:
+
+```
+library("dccvalidator")
+run_app()
```

Comment:
This would use "default" as the config, correct? If so, we may want to specify that they should make their config be the default, as opposed to naming it as the instructions (instruction 5).

Reply:
Yes -- ultimately I'd like to make the configuration file, and perhaps which configuration to use, a parameter to to the app, but for now it depends on a config file in the same directory with a default config.

Reply:
Alrighty. Sounds good.